<launch>
  <node name="keyboard_control" pkg="keyboard_control" type="keyboard_control.py"/>
  <node name="orientation_gain_control" pkg="motor_control" type="PIDGainControlGUI.py" respawn="true" args="/motor_control/OrientationGainControl" output="screen"/>
  <!--node name="distance_gain_control" pkg="motor_control" type="PIDGainControlGUI.py" respawn="true" args="/motor_control/DistanceGainControl" output="screen"/-->
  <!--node name="plot_speed_left" pkg="rqt_plot" type="rqt_plot" args="/motion/Odometry/theta"/-->
  <node name="odometry_plot" pkg="rxtools" type="rxplot" args="/motion/Odometry/theta"/>
  <!--node name="plot_speed_right" pkg="rxtools" type="rxplot" args="/motion/SpeedReference/W2 /motion/Speed/W2"/-->
</launch>
